linkerd/app/outbound/src/endpoint.rs

```diff
@@ -68,64 +67,52 @@ impl<E> Outbound<E> {
     }
 
     /// Pushes a layer that checks if a discovered service profile contains an
-    /// endpoint override, and forwards directly to that endpoint (bypassing the
-    /// current stack) if one exists.
-    pub fn push_endpoint_override<T, O, R>(
+    /// endpoint, and forwards directly to that endpoint (bypassing the current
+    /// stack) if one exists.
+    pub fn push_profile_endpoint<T, E, ESvc, NSvc, Req>(
```

Comment:
:+1: the "override" terminology was just my first thought when trying to describe this — it's probably not necessary. 

I do think it _might_ be nice to have a consistent language for layer functions that introduce a "branch", like this one and `push_unwrap_logical`, like `or_profile_endpoint` or something? but we can do that later.

Reply:
I think the "override" language might also show up in a test someplace — but I can fix that.